exchanges/kraken/kraken_test.go

```diff
@@ -543,6 +543,27 @@ func TestCancelExchangeOrder(t *testing.T) {
 	}
 }
 
+// TestCancelExchangeOrder wrapper test
+func TestCancelBatchExchangeOrder(t *testing.T) {
+	if areTestAPIKeysSet() && !canManipulateRealOrders {
+		t.Skip("API keys set, canManipulateRealOrders false, skipping test")
+	}
+
+	var ordersCancellation []*order.Cancel
+	ordersCancellation = append(ordersCancellation, &order.Cancel{
+		ID:        "OGEX6P-B5Q74-IGZ72R,OGEX6P-B5Q74-IGZ722",
+		AssetType: asset.Spot,
```

Comment:
This also needs a currency pair set due to the use of `Validate()` in `CancelBatchOrders`

Reply:
OK, I have added pair here, but I don't know which pair you are using for this order